config/sidekiq_scheduler.yml

```diff
@@ -24,7 +24,7 @@ EducationForm::Create10203SpoolSubmissionsReport:
   description: "Send the daily 10203 report to the call center about spool file submissions"
 
 EducationForm::Create10203ApplicantDecisionLetters:
-  cron: "45 4 * * MON-FRI America/New_York"
+  every: '2m'
```

Comment:
This is going from running M-F once a day to running every 2 minutes? That seems like quite a jump. Just want to verify that this is intended.

Reply:
Updating frequency for UAT testing. This change will be reverted after UAT testing